ribbon/encode.py

```diff
@@ -0,0 +1,388 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+#----------------------------------------------------------------------------
+# Created By: Steven (steven@ribbon.finance)
+# Created Date: 04/04/2022
+# version ='0.01'
+# ---------------------------------------------------------------------------
+''' Module to encode message '''
+# ---------------------------------------------------------------------------
+
+# ---------------------------------------------------------------------------
+# Imports
+# ---------------------------------------------------------------------------
+from web3 import Web3
+import re
+from utils import *
+
+# ---------------------------------------------------------------------------
+# Constants
+# ---------------------------------------------------------------------------
+DOMAIN_FIELD_NAMES = [
+  'name', 
+  'version', 
+  'chainId', 
+  'verifyingContract', 
+  'salt'
+]
+DOMAIN_FIELD_TYPES = {
+    'name': 'string',
+    'version': 'string',
+    'chainId': 'uint256',
+    'verifyingContract': 'address',
+    'salt': 'bytes32'
+}
+
+HEX_TRUE = hexZeroPad(Web3.toHex(1), 32)
+HEX_FALSE = hexZeroPad(Web3.toHex(0), 32)
+
+# ---------------------------------------------------------------------------
+# Helper Functions
+# ---------------------------------------------------------------------------
+def uintEncoder(type: str) -> object:
```

Comment:
If possible to change now, the function names and method names in classes in python follow the [snake case](https://en.wikipedia.org/wiki/Snake_case), just like the variable names.

So nice to see type hints :)

Reply:
Done